app/upgrades/v2-beta.1/constants.go

```diff
@@ -0,0 +1,50 @@
+package v2_beta1
+
+import (
+	"github.com/UnUniFi/chain/app/upgrades"
+	store "github.com/cosmos/cosmos-sdk/store/types"
+)
+
+const UpgradeName string = "v2-beta.1"
```

Comment:
"v2-beta.1" sounds actually weird.
just "v2" or "v1-beta.4" sounds better. 
cc: @mkXultra , @KimuraYu45z

Reply:
thank you!
fix upgrede name